### ISSUE TITLE
dotnet-add-package : Typo in description implies support for multiple packages

### ISSUE THREAD

**Issue by spboyer** [dotnet/cli#5740]
## Steps to reproduce
```
dotnet add package --help
.NET Add Package reference Command

Usage: dotnet add <PROJECT> package [arguments] [options]

Arguments:
  <PACKAGE_NAME>  Package references to add
  <PROJECT>       The project file to operate on. If a file is not specified, the command will search the current directory for one.
```

`dotnet add package Newtonsoft.Json Swagger`

## Expected  behavior
Adds Newtonsoft.Json and Swagger NuGet Package to project

## Actual behavior
Error:
“Please specify one package reference to add.”

## Environment data
```
Product Information:
 Version:            1.0.0-rc4-004777
 Commit SHA-1 hash:  18c01886cd

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/1.0.0-rc4-004777
```

**Comment by livarcocc** [comment]
This will be fixed by https://github.com/dotnet/cli/pull/5875.

**Comment by mairaw** [comment]
@spboyer actually fixed this one with #5741. Closing.
